dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -197,6 +197,11 @@ bool FramesDecoder::ReadRegularFrame(uint8_t *data, bool copy_to_output) {
 
 void FramesDecoder::Reset() {
   next_frame_idx_ = 0;
+
+  if (flush_state_) {
+    flush_state_ = false;
+  }
+
```

Comment:
While improving tests coverage I found this small bug in the frames decoder - `flush_state_` flag was not cleared while decoder was reset.